test/test_data.c

```diff
@@ -173,8 +175,10 @@ struct ts_data_object data_objects[] = {
 
     TS_FUNCTION(0x5001, "x-dummy", &dummy, ID_RPC, TS_ANY_RW),
 
+#if TS_64BIT_TYPES_SUPPORT
```

Comment:
Test should cover all aspects and fail if not all aspects are tested. Is this to re-use in examples?

Reply:
The tests cover all cases because the flags are switched on.

PlatformIO UNITY tests:
https://github.com/ThingSet/thingset-device-library/blob/1eaa305a9f1873f676b77b8dd0f9d34f1944ecd1/platformio.ini#L36-L38

Zephyr ztests:
https://github.com/ThingSet/thingset-device-library/blob/1eaa305a9f1873f676b77b8dd0f9d34f1944ecd1/zephyr/tests/prj.conf#L5-L8